recipes/deepseek/DeepSeek-R1-Distill-Qwen-7B/grpo/config_base_math_smalllr.yaml

```
@@ -40,6 +40,12 @@ per_device_train_batch_size: 1
 push_to_hub: true
 report_to:
 - wandb
+reward_funcs:
+  - accuracy
+  - format
+reward_weights:
+  - 1.0
+  - 0.5
```

```suggestion
  - 1.0
```
While I agree with the weights you proposed, can we have the values in the recipes match the original values, otherwise we would be unable to reproduce their exact results.